go.mod

```diff
@@ -0,0 +1,48 @@
+module github.com/kiegroup/kogito-cloud-operator
+
+require (
+	github.com/NYTimes/gziphandler v1.0.1 // indirect
+	github.com/RHsyseng/operator-utils v0.0.0-20190807020041-5344a0f594b8
+	github.com/ghodss/yaml v1.0.0
+	github.com/go-logr/logr v0.1.0
+	github.com/gobuffalo/packr/v2 v2.5.2
+	github.com/google/uuid v1.1.1
+	github.com/mitchellh/go-homedir v1.1.0
+	github.com/openshift/api v0.0.0-00010101000000-000000000000
+	github.com/openshift/client-go v0.0.0-00010101000000-000000000000
+	github.com/operator-framework/operator-sdk v0.9.1-0.20190712203509-e1d904fa80a4
+	github.com/spf13/cobra v0.0.5
+	github.com/spf13/pflag v1.0.3
+	github.com/spf13/viper v1.3.2
+	github.com/stretchr/testify v1.3.0
+	go.uber.org/zap v1.9.1
+	gopkg.in/yaml.v2 v2.2.2
+	k8s.io/api v0.0.0-20190612125737-db0771252981
+	k8s.io/apimachinery v0.0.0-20190612125636-6a5db36e93ad
+	k8s.io/client-go v11.0.0+incompatible
+	k8s.io/kube-openapi v0.0.0-20190603182131-db7b694dc208 // indirect
+	sigs.k8s.io/controller-runtime v0.1.12
+	sigs.k8s.io/controller-tools v0.1.10
+)
+
+// Pinned to kubernetes-1.13.4
+replace (
+	k8s.io/api => k8s.io/api v0.0.0-20190222213804-5cb15d344471
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.0.0-20190228180357-d002e88f6236
+	k8s.io/apimachinery => k8s.io/apimachinery v0.0.0-20190221213512-86fb29eff628
+	k8s.io/client-go => k8s.io/client-go v0.0.0-20190228174230-b40b2a5939e4
+)
+
+replace (
+	github.com/coreos/prometheus-operator => github.com/coreos/prometheus-operator v0.29.0
+	k8s.io/kube-state-metrics => k8s.io/kube-state-metrics v1.6.0
+	sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.1.12
+	sigs.k8s.io/controller-tools => sigs.k8s.io/controller-tools v0.1.11-0.20190411181648-9d55346c2bde
+)
+
+replace github.com/operator-framework/operator-sdk => github.com/operator-framework/operator-sdk v0.9.0
+
+replace (
+	github.com/openshift/api => github.com/openshift/api v3.9.1-0.20190814194116-a94e914914f4+incompatible
+	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20190813201236-5a5508328169
```

Comment:
This release is from branch `release-4.3`? If the unit tests are working, I'm assuming is a version compatible with the fake client from Kubernetes.

Reply:
These versions are mapped from `release-4.3`. I specified `release-4.3` manually in the file and the versions are converted after executing `go mod vendor`.